core/src/com/group66/game/cannon/Cannon.java

```diff
@@ -30,34 +57,74 @@ else if (angle <= 0)
 			angle = 1;
 	}
 	
+	/**
+	 * Sets the angle of the cannon.
+	 *
+	 * @param angle the new angle
+	 */
 	public void setAngle(float angle) {
 		this.angle = angle;
```

Comment:
I was thinking about that, but the setAngle is not used within the game, If anything I would like to change the checks for the maximum angle to GameScreen as the cannon itself should now about its angle limitations. (again going back to the point that classes like cannon should be implemented as separate entities and are used by the game, not know of the game).

So I think move the check to GameScreen, you?